### PR TITLE
fixed metric name resolver; added test for it

### PR DESCRIPTION
**Pull request by ankitcha**

### CODE REVIEW

File: src/main/scala/mix/kafka/monitor/LibratoOffsetReporter.scala [hunk 1]
```diff
@@ -73,3 +65,16 @@ class LibratoOffsetReporter(metrics: MetricRegistry,
...
+
+object LibratoOffsetReporter {
+  def getMetricName(offsetInfo: OffsetInfo): String = {
+    val nameComponents = Seq (
+      offsetInfo.group,
+      offsetInfo.topic,
+      offsetInfo.partition.toString,
+      "lag"
+    )
+
```
Comment (dkjhanitt) [inline src/main/scala/mix/kafka/monitor/LibratoOffsetReporter.scala:77]: I believe we do not need to ensure that metric name should be in `lowercase` ?

**Review by dkjhanitt — DISMISSED** [review]

File: src/main/scala/mix/kafka/monitor/LibratoOffsetReporter.scala (hunk 1 above)
Comment (ankitcha) [reply src/main/scala/mix/kafka/monitor/LibratoOffsetReporter.scala:77]: Good idea - even if its allowed - i think it is still good to convert everything to lowercase for consistency and readability

Comment (ankitcha) [reply src/main/scala/mix/kafka/monitor/LibratoOffsetReporter.scala:77]: done!

Comment (dkjhanitt) [reply src/main/scala/mix/kafka/monitor/LibratoOffsetReporter.scala:77]: 😎

**Review by dkjhanitt — APPROVED** [review]

**Pull request merged**